example/my-app/spago.yaml

```diff
@@ -0,0 +1,11 @@
+package:
+  name: my-app
+  run:
+    main: App.Main
```

Comment:
Once these stanzas become stable (cc: @f-f) then we could parse them using the `from-yaml` utility. On the other hand, I'm not sure whether we should parse the bundling part of the spago config, because there's no guarantee anyone's going to be using esbuild.